### PR TITLE
themes: Add onedarker

### PR DESCRIPTION
**Pull request by nuid32**
Dark theme partially copyed from LunarVim
onedarker
onedarker0

### CODE REVIEW

File: runtime/themes/onedarker.toml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+"variable.other.member" = { fg = "blue" }
+
+"markup.heading" = { fg = "red" }
+"markup.raw.inline" = { fg = "green" }
+"markup.bold" = { fg = "gold", modifiers = ["bold"] }
+"markup.italic" = { fg = "purple", modifiers = ["italic"] }
+"markup.list" = { fg = "red" }
+"markup.quote" = { fg = "yellow" }
+"markup.link.url" = { fg = "cyan", modifiers = ["underlined"]}
+"markup.link.text" = { fg = "purple" }
```
Comment (AlexanderBrevig) [inline runtime/themes/onedarker.toml:36]: `markup.link.label` missing

File: runtime/themes/onedarker.toml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+"special" = { fg = "blue" }
+"string" = { fg = "green" }
+"type" = { fg = "yellow" }
+# "variable" = { fg = "blue" }
+"variable.builtin" = { fg = "red" }
+"variable.parameter" = { fg = "red" }
+"variable.other.member" = { fg = "blue" }
+
+"markup.heading" = { fg = "red" }
+"markup.raw.inline" = { fg = "green" }
```
Comment (AlexanderBrevig) [inline runtime/themes/onedarker.toml:30]: `markup.raw.block` missing

**Review by AlexanderBrevig — CHANGES_REQUESTED** [review]
Please attach a screenshot to your description, and maybe a sentence or two about the theme?

**Review by AlexanderBrevig — APPROVED** [review]
LGTM

File: runtime/themes/onedarker.toml [hunk 3]
```diff
@@ -0,0 +1,97 @@
+# Author : Gokul Soumya <gokulps15@gmail.com>
```
Comment (the-mikedavis) [inline runtime/themes/onedarker.toml:1]: This should be replaced

Comment (Ziaft) [reply runtime/themes/onedarker.toml:1]: Oh,  shoot, thank you

**Review by the-mikedavis — APPROVED** [review]

**Comment by the-mikedavis** [comment]
Could you add a screenshot of this to the themes wiki? https://github.com/helix-editor/helix/wiki/Themes

**Comment by nuid32** [comment]
ok

**Pull request merged**